config/src/test/scala/com/typesafe/config/impl/TokenizerTest.scala

```diff
@@ -33,75 +38,95 @@ class TokenizerTest extends TestUtils {
         // all token types with no spaces (not sure JSON spec wants this to work,
         // but spec is unclear to me when spaces are required, and banning them
         // is actually extra work).
+        val source = """,:=}{][+="foo"""" + "\"\"\"bar\"\"\"" + """true3.14false42null${a.b}${?x.y}${"c.d"}""" + "\n"
         val expected = List(Tokens.START, Tokens.COMMA, Tokens.COLON, Tokens.EQUALS, Tokens.CLOSE_CURLY,
             Tokens.OPEN_CURLY, Tokens.CLOSE_SQUARE, Tokens.OPEN_SQUARE, Tokens.PLUS_EQUALS, tokenString("foo"),
             tokenString("bar"), tokenTrue, tokenDouble(3.14), tokenFalse,
             tokenLong(42), tokenNull, tokenSubstitution(tokenUnquoted("a.b")),
             tokenOptionalSubstitution(tokenUnquoted("x.y")),
             tokenKeySubstitution("c.d"), tokenLine(1), Tokens.END)
-        assertEquals(expected, tokenizeAsList(""",:=}{][+="foo"""" + "\"\"\"bar\"\"\"" + """true3.14false42null${a.b}${?x.y}${"c.d"}""" + "\n"))
+        assertEquals(expected, tokenizeAsList(source))
+        assertEquals(source, tokenizeAsString(source))
```

Comment:
A bunch of these tests could just be using the tokenizerTest method I guess but aren't? Not sure why they weren't already, some historical accident unless I'm missing something.